tla/consensus/MCAliases.tla

```
@@ -83,10 +84,10 @@ DebugAliasVars ==
     ]
 
 DebugAlias ==
-    [ _format |-> B[srv] \o "/\\[0m %1$s = %2$s\n" ]
-      @@
-    DebugAliasAggregates
-      @@
+    \*[ _format |-> B[srv] \o "/\\[0m %1$s = %2$s\n" ]
```

These seem to currently be broken, and when the alias is broken it is just silently ignored.